pkg/scanner/enrich_codeanalysis.go

```diff
@@ -0,0 +1,58 @@
+package scanner
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/safedep/vet/pkg/code"
+	"github.com/safedep/vet/pkg/models"
+)
+
+type CodeAnalysisEnricherConfig struct {
+	ReaderRepository        code.ReaderRepository
```

Comment:
`ReaderRepository` is not a config I think. Its more of a dependency to `NewCodeAnalysisEnricher`. I think `vet` probably have some inconsistency between config and dependency but lets get it right here.

`ReaderRepository` can be passed as an arg to `NewCodeAnalysisEnricher` which explicitly states that `ReaderRepository` is a dependency for this enricher along with config.

Reply:
got it, my bad